cadl-tests/cadl/special-headers.cadl

```diff
@@ -23,30 +25,23 @@ model Resource {
   type: string;
 }
 
-model SpecialHeaderParameters {
-  @header("Repeatability-Request-ID") requestId?: string;
-  @header("Repeatability-First-Sent") firstSent?: zonedDateTime;
-}
-
-model Customizations {
-  parameters: SpecialHeaderParameters;
-}
+interface ResourceOperations
+  extends Azure.Core.ResourceOperations<SupportsRepeatableRequests & NoConditionalRequests & NoClientRequestId> {}
```

Comment:
Use SupportsRepeatableRequests trait

Reply:
https://github.com/Azure/cadl-azure/blob/main/packages/cadl-azure-core/lib/traits.cadl#L143-L156

Reply:
I checked trait related [doc](https://azure.github.io/cadl-azure/docs/getstarted/azure-core/step09). From my understanding, `SupportsRepeatableRequests` trait is used to make sure `repeatability-request-id` and `repeatability-first-sent` exist in request and response header, in other word, we will treat these two header params as required i.e. putting them to the method signature. Please correct me if I am wrong.

Reply:
This 2 param is special. They are generated by SDK instead (a random GUID and current time) for this specific purpose https://docs.oasis-open.org/odata/repeatable-requests/v1.0/cs01/repeatable-requests-v1.0-cs01.html
(hence this is `special-headers.cadl`. this test is not really relates to trait, it is to test that these header is indeed provided by SDK, not suer)